src/gui/optionsdialog.ui

```diff
@@ -707,6 +707,13 @@
               </layout>
              </widget>
             </item>
+            <item>
+             <widget class="QCheckBox" name="checkBoxPerformanceWarnings">
+              <property name="text">
+               <string>Log performance warnings</string>
```

Comment:
Do you really believe that this is one of the commonly used options, and not an advanced one?

Reply:
I reckon (for qbt section) only if the option is experimental or very obscure should be an advanced option and this one is neither of them.